### ISSUE TITLE
Failed to decode downloaded font

### ISSUE THREAD

**Issue by balkarov** [frankwallis/WebpackAspnetMiddleware#11]
Hello. 
Thank you for your library. 

When I use your middleware I get error for loading fonts
```
Failed to decode downloaded font: http://localhost:12496/bundle/d0c6e2dcac826fe1c798bf6a4986b0f2.woff
OTS parsing error: incorrect file size in WOFF header
```

You can try use font-awesome in webpack

PS: Without middleware all works

**Comment by lainiwakurafan** [comment]
@balkarov, hey, yeah! I have the same issue...

![41517a7fb34af7ce505f7c9bcdfb7dea](https://user-images.githubusercontent.com/24371248/31823286-5372fa14-b5b4-11e7-902a-89d80cc6d022.png)

**Comment by balkarov** [comment]
@lainiwakurafan I solved my problem.
I just change to native library 😄 
UseWebpackDevMiddleware from [Microsoft.AspNetCore.SpaServices](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.builder.webpackdevmiddleware.usewebpackdevmiddleware?view=aspnetcore-2.0)

I advise use it. More information on [JavaScriptServices](https://github.com/aspnet/JavaScriptServices)

**Comment by frankwallis** [comment]
Thank you for raising this, it should now be fixed in 2.1.0. It will also require upgrading the webpack-aspnet-middleware npm package to 2.1.0 as well.
